src/prognostic_equations/implicit/implicit_tendency.jl

```diff
@@ -77,26 +119,15 @@ vertical_advection!(ᶜρχₜ, ᶠu³, ᶜχ, ::Val{:third_order}) =
     @. ᶜρχₜ -= ᶜadvdivᵥ(ᶠupwind3(ᶠu³, ᶜχ)) - ᶜχ * ᶜadvdivᵥ(ᶠu³)
 
 function implicit_vertical_advection_tendency!(Yₜ, Y, p, t, colidx)
-    (; energy_upwinding, density_upwinding) = p.atmos.numerics
-    (; tracer_upwinding, precip_upwinding) = p.atmos.numerics
```

Comment:
We should not be removing `precip_upwiding` from this tendency. Precipitation always needs to be upwinded in order for microphysics to work correctly.